### ISSUE TITLE
Can't use the extension

### ISSUE THREAD

**Issue by e2raptor** [pajoma/vscode-journal#10]
I've installed the extension and configured in the settings the "journal.base" to this "/Users/eduardo/Dropbox/journal". When I try to use for example the following:

```journal:today ``` 

I get a VS error that says: 

```Running the contributed command:'journal.today' failed.```

 Same thing if I try with ```journal:tomorrow``` or ```journal:yesterday```. 

Also if I press Cmd+Shift+J and then type 0, +1, -1, monday, anything, then nothing happens, same thing with Quick Memo. Actually the only two commands that work are: journal: Notes (which creates a new note and when I save it, it appears in the folder) and journal: Open the Journal (which perfectly opens the journal folder I've configured)

I've checked folder permissions but doesn 't seems to be the problem. I'm using MacOs El Capitan, VSC Version 1.8.1 (1.8.1), and the extension version is 0.4.0 (The only one that vs downloads) 

Perhaps is something I'm missing or that I haven't configured, maybe with the VS. I'd really love to use the extension since I was using something similar with Vim and I'm really used to it. Thanks in advance.

**Comment by e2raptor** [comment]
Well, I found what the problem is. I cloned the repo and debugged  the extension by myself. I found the problem in the out/src/util/conf.js file:

```return (locale.length == 0) ? locale : 'en-US';```

It turns out that my locale was empty string "" and thus that was returning. I modified to:

```return (locale.length == 0) ? 'en-US' : locale;```

And now it works. Correct me if I'm wrong but,  if no locale is found I think it should return en-US instead of the empty string.
